packages/ia-zendesk-help-widget/test/zendesk-help-widget.test.js

```diff
@@ -0,0 +1,89 @@
+import {
+  html,
+  fixture,
+  expect
+} from '@open-wc/testing';
+import '../src/ia-zendesk-help-widget';
+
+const component = (properties = {
+  widgetSrc: 'https://static.zdassets.com/ekr/snippet.js?key=685f6dc4-48c5-411f-8463-cc6dd50abe2d',
```

Comment:
Is this making a request to the live zen desk site during the unit test? If so, I would recommend replacing it with a local mock javascript file that simulates the part of the zen desk widget that you are interacting with. We don't really want to be making network requests during unit testing.

If you add the mock widget to this directory, you can set your `widgetSrc` to `/base/test/mock-zen-desk.js` and it will find it. You can see some examples in [the unit tests for the lazy loader service](https://github.com/internetarchive/iaux-lazy-loader-service/blob/master/test/lazy-loader-service.test.ts#L5-L13).